docs/index.md

```diff
@@ -39,17 +42,29 @@ The image below shows the timing around how the Mobi Planner turns Bookings and
 - **Master Data:** [Master Data](#master-data) is relatively static data that includes information about physical places and the Business Rules that should apply to relevant Bookings during planning. The client can provide REST APIs for Mobi to call regularly to update Master Data, and can optionally send updates via an AWS Data Stream as well with specific information about what Bookings should be updated. These mechanisms are described further in [Sending Master Data](#sending-master-data).
+  - Jamie: This is a totally open question...  There are many ways we could do this (e.g. should we pull data from them,
+    or should they push data to us?).  Here I think we have the opportunity to see what type of mechanism will be
+    easiest for them, and to come up with a generalized mechanism around that.
 
 ## Regular Planning Overview
 
 1. When a Booking comes in via the AWS Kinesis Data Stream, it gets ingested but not planned until the **Regular Planning Window** begins for the relevant Destination and date. For most Destinations, the Regular Planning Window begins 7 days before the date of travel and ends 24 hours before the time of travel.
 2. Every 5 minutes, the Mobi Planner runs Regular Planning. First, it checks to see if any Bookings within their Regular Planning Window are new, have been updated, have been removed, or have had updates to their corresponding Flight. Then, it plans the changed Bookings and any other Bookings that could potentially be on the same Trip (e.g. Bookings in the same Destination on the same date of travel).
+   1. Jamie: Just a nitpick--I don't think we want to call this the "Mobi Planner," as this usually refers to something
+      else.  Is there a distinction you're trying to make between "Mobi Planner" and "Continous Planning System?"
 3. The Mobi Planner uses a combination of cutting-edge AI algorithms and planning methods based on insight from human operators to quickly and efficiently optimize a set of Trips that satisfies the client's Business Rules.
 4. The Continuous Planning System computes the timing for each stop within the trip based on Mobi's internal routing engine, then validates that the solution passes a set of criteria including the client's Business Rules (e.g. passengers don't spend more than the maximum time waiting at the airport)
+   1. Another nitpick--I wouldn't phrase this as two separate steps (_i.e._ #1 we compute the trip, #2 we do validation).
+      We really refer to the business rules throughout the planning process.  IMO specifying this as two separate steps 
+      makes us sounds less sophisticated than we really are.
```

Comment:
I think this is a good point. Maybe we could call it like a final quality control or something? If we want to emphasize something along those lines here